services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouterHttpRequestStats.java

```diff
@@ -129,9 +129,10 @@ public class RouterHttpRequestStats extends AbstractVeniceHttpStats {
   private final Sensor errorRetryAttemptTriggeredByPendingRequestCheckSensor;
 
   private final String systemStoreName;
-  private final Attributes commonMetricDimensions;
-  private final boolean emitOpenTelemetryMetrics;
-  private final VeniceOpenTelemetryMetricNamingFormat openTelemetryMetricFormat;
+  private Attributes commonMetricDimensions = null;
+  private boolean emitOpenTelemetryMetrics = false;
+  private VeniceOpenTelemetryMetricNamingFormat openTelemetryMetricFormat =
+      VeniceOpenTelemetryMetricNamingFormat.getDefaultFormat();
```

Comment:
I would prefer that we keep these field final. Is there a reason to change this?

Reply:
I modified this to cut a couple of lines in the code, but having these class fields as final makes better sense. Changed it back.